jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -92,6 +95,8 @@
    * @throws IOException if an error occurs creating the container builder
    * @throws InvalidWorkingDirectoryException if the working directory specified for the build is
    *     invalid
+   * @throws InvalidPlatformConfigurationException if there exists a plaformConfiguration with a
+   *     null architectureName or osName
```

Comment:
I think this sounds better since a platformConfig is the one missing values .

` if there exists a platformConfiguration in the platforms list that is missing required fields or has invalid values `

Reply:
Sounds good. I'd go with `... exists a {@link PlatformConfiguration} ...` in that case.